### ISSUE TITLE
Is RMStore still usable ?

### ISSUE THREAD

**Issue by meoz** [robotmedia/RMStore#207]
I see the library is not been updated for more than a year. I'm wondering if it's still reliable to use in it's current state in an app with autorenewable-subscriptions ? 
If not is there a fork or alternative which is been worked on ?

**Comment by inPhilly** [comment]
I would also be interested in this. If not, is there a better library for use in an objective c app?

**Comment by DoctorG** [comment]
Is there a reason to believe it isn't working?

**Comment by iwasrobbed** [comment]
Have a look at https://github.com/bizz84/SwiftyStoreKit used in conjunction with https://github.com/tikhop/TPInAppReceipt

**Comment by tblanchard** [comment]
Read the code.  There's not much of it and what there is looks pretty clean.

As to SwiftyStoreKit - not everyone wants to take on the overhead of the swift treadmill.

**Comment by vince4** [comment]
RMStore project owner @hpique has made no public commit on github for 1 year. 
I loved this project, but with new features on StoreKit, I think RMStore should be deprecated. 

Sorry :-'(

**Comment by inPhilly** [comment]
So it is basically still working with the latest iOS (11) - even though new features have not been implemented?

**Comment by vince4** [comment]
Yes, it is still working with iOS 11, and Promoting In App Purchase (new feature) is not implemented.
